Libraries/NativeModules/specs/NativeRedBox.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+import {Platform} from 'react-native';
+
+export interface Spec extends TurboModule {
+  +setExtraData: (extraData: Object, identifier: string) => void;
+  +dismiss: () => void;
+}
+
+export default (Platform.OS === 'ios'
+  ? TurboModuleRegistry.getEnforcing<Spec>('RedBox')
+  : null);
```

Comment:
Shouldn't we use `undefined` here instead?

Reply:
hmm good question, I didn't see anything about using it specifically. But it would be nice to have a consensus.

Reply:
I saw some of the specific platforms PRs using `undefined` and others using `null`.
I don't have a strong opinion on that, but my preference is to go with `undefined` what denotes that this Native Module "was not defined" for this platform.

Anyway, I can update my PRs if we decide to go with `null`.

Reply:
We should actually be using get not get enforcing

Reply:
Yeah, I'll update it before I import/land to FB